README.md

```diff
@@ -1,2 +1,40 @@
 # sshd-docker
 Docker image with SSH daemon installed
+
+# Usage
+This container is meant to be used to expose host ports to containers running in Docker.
+When using testcontainers in unit tests, refer to the documentation of testcontainers to find out how to use it:
+- Java: https://java.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
+- .NET: https://dotnet.testcontainers.org/api/create_docker_network/#exposing-host-ports-to-the-container
+- node.js: https://node.testcontainers.org/features/networking/#expose-host-ports-to-container
+- Go: https://golang.testcontainers.org/features/networking/#exposing-host-ports-to-the-container
+
+Alternatively, you can start the container yourself:
+```yaml
```

Comment:
This LGTM, although I'm trying to reproduce this scenario with compose and the python http server locally without success. Can you double check it works 🙏 ?

Once verified, we can merge this, thanks!

Reply:
so after the port fix this is how it looks like on my machine:
![image](https://github.com/user-attachments/assets/5b426420-4b1e-44fa-a150-bdb79905e9e3)


You might get a few connection refused at the very start while the server is not running, but once it is started you should see requests on the python server and the responses in the docker logs

Reply:
I also published the folder here: https://github.com/StefanLobbenmeierObjego/testcontainers-sshd-example but should be the same content as here

Reply:
Cool, thanks for sharing! I cloned the repro project and it worked! I'm sure I did something wrong with my previous setup, so than you for taking the time to explaining it to me.

Yeah, this LGTM!